roborazzi/src/main/java/com/github/takahirom/roborazzi/RobolectricDeviceQualifiers.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.takahirom.roborazzi
+
+object RobolectricDeviceQualifiers {
+  // Generated by [GenerateQualifiersTest.kt]
+  // Data from: AOSP https://android.googlesource.com/platform/tools/base/+/mirror-goog-studio-master-dev/sdklib/src/main/java/com/android/sdklib/devices
+// FROM: tv.xml
+  const val AndroidTV1080p = "w960dp-h540dp-xlarge-long-notround-television-xhdpi-keyshidden-dpad"
+  const val AndroidTV720p = "w962dp-h541dp-xlarge-long-notround-television-tvdpi-keyshidden-dpad"
+
+  // FROM: devices.xml
+// FROM: wear.xml
+  const val WearOSSquare = "w187dp-h187dp-small-long-round-watch-hdpi-keyshidden-nonav"
```

Comment:
I'll put up a PR to replace WearOSRound and WearOSRoundChin with WearOSLargeRound and WearOSSmallRound.

Reply:
Done in https://github.com/takahirom/roborazzi/pull/56